lib/web/apiserver_test.go

```diff
@@ -960,15 +961,20 @@ func (s *WebSuite) TestResizeTerminal(c *C) {
 	// because resize events are not sent back to the originator.
 	params, err := session.NewTerminalParamsFromInt(300, 120)
 	c.Assert(err, IsNil)
-	websocket.JSON.Send(ws2, eventEnvelope{
-		Type: defaults.ResizeRequestEnvelopeType,
-		Payload: events.EventFields{
-			events.EventType:      events.ResizeEvent,
-			events.EventNamespace: defaults.Namespace,
-			events.SessionEventID: sid.String(),
-			events.TerminalSize:   params.Serialize(),
-		},
+	data, err := json.Marshal(events.EventFields{
```

Comment:
why would you marshal json and then marshal it in protobuf? would not it be easier to marshal it in protobuf? may be  write this as a next improvement or something

Reply:
I did this so we can re-use our marshal/unmarshal logic for events. Otherwise we'd have to write something that's one off.